src/app/settings/components/settings/settings.component.html

```diff
@@ -0,0 +1,6 @@
+<sofie-header></sofie-header>
+
+<mat-drawer-container class="c-settings">
```

Comment:
Same comment as for the `mat-toolbar` :)

Reply:
Will try to remove it for now.